src/main/java/frc/robot/commands/HalfSpeed.java

```diff
@@ -1,60 +1,35 @@
-// RobotBuilder Version: 3.1
-//
-// This file was generated by RobotBuilder. It contains sections of
-// code that are automatically generated and assigned by robotbuilder.
-// These sections will be updated in the future when you export to
-// Java from RobotBuilder. Do not put any code or make any change in
-// the blocks indicating autogenerated code or it will be lost on an
-// update. Deleting the comments indicating the section will prevent
-// it from being updated in the future.
-
-
 package frc.robot.commands;
-import edu.wpi.first.wpilibj2.command.InstantCommand;
 
-// BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=IMPORTS
+import frc.robot.RobotContainer;
 import frc.robot.subsystems.Drivetrain;
 
-    // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=IMPORTS
-
-/**
- *
- */
-public class HalfSpeed extends InstantCommand {
-
-    // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=VARIABLE_DECLARATIONS
-        private final Drivetrain m_drivetrain;
- 
-    // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=VARIABLE_DECLARATIONS
-
-    // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTRUCTORS
-
+public class HalfSpeed extends TankDrive {
```

Comment:
Let's make this an instant command that changes a parameter internal to the drivetrain subsystem.